.github/workflows/action.yml

```diff
@@ -0,0 +1,34 @@
+name: Satellite-Upgrade
+
+on:
+  pull_request:
```

Comment:
Questions:
1. Don't we need `push:` logic for satellite-upgrade?

2. are you planning for keeping Travis config for some more time in this repo? can be removed in the same PR.

Reply:
Push is not required here, I will raise separate PR for Travis removal not.

Reply:
Okay, LGTM otherwise 👍